src/components/VCSMenu.tsx

```diff
@@ -174,7 +177,7 @@ export class VCSMenu extends React.Component<VCSMenuProps, VCSMenuState> {
    */
   plot() {
     if (this.state.selectedVariables.length == 0) {
-      this.props.inject("# Please select a variable from the left panel");
+      alert("Please select a variable from the left panel");
```

Comment:
This is an example of where we should be useing the jupyter Dialog, but its fine for now.

Reply:
I added a function to notebook_utils which will create a JupyterLab dialog message and have replaced all the alert statements with it.